### PR TITLE
added Logger

### PR DESCRIPTION
**Pull request by suiluj482**
daily rotating logs

in dev logging human readable to diffrent file

logger with labels for db and auth

### CODE REVIEW

File: server/utils/loggers.ts
Comment (niklhut) [inline server/utils/loggers.ts:41]: I think we should also log to the console in production since it is helpful to see the live output in docker. So we should discuss this before merging, I am just blocking this for now.

File: server/utils/loggers.ts
Comment (niklhut) [inline server/utils/loggers.ts:28]: Ah and can you remove these unused code comments

File: server/utils/loggers.ts
Comment (niklhut) [inline server/utils/loggers.ts:33]: This one too I guess, as long as it is only outcommented code

**Review by niklhut — CHANGES_REQUESTED** [review]

File: server/utils/loggers.ts
Comment (suiluj482) [reply server/utils/loggers.ts:33]: i thought this might be a nice reminder its an option in the furute

**Review by niklhut — APPROVED** [review]
LGTM, fixes #25

**Pull request merged**
